tests/wpt/include.ini

```diff
@@ -39,6 +39,8 @@ skip: true
         skip: false
 [cors]
   skip: false
+[webgl]
```

Comment:
They were disabled again by @Ms2ger, since they caused a lot of intermittents :/